web/tests/Makefile

```diff
@@ -44,97 +47,96 @@ PYLINT_TEST_CMD = pylint ./bin/codechecker-web-version \
   --disable=all \
   --enable=logging-format-interpolation,old-style-class
 
-pylint: venv_dev
-	$(ACTIVATE_DEV_VENV) && $(PYLINT_TEST_CMD)
+pylint:
+	 $(PYLINT_TEST_CMD)
 
+pylint_in_env:
+	$(ACTIVATE_DEV_VENV) && $(PYLINT_TEST_CMD)
+ 
 CODECHECKER_CMD = $(BUILD_DIR)/CodeChecker/bin/CodeChecker
 SHUTDOWN_SERVER_CMD = echo "Shutting down server..."; \
-    HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l; \
-    HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server \
-			--config-directory $(CC_TEST_WORKSPACE_ROOT) \
-	    --port `cat "$(CC_TEST_WORKSPACE_ROOT)/serverport"` --stop; \
-    rm -f "$(CC_TEST_WORKSPACE_ROOT)/serverport"; \
-    HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l
+  HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l; \
+  HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server \
+  --config-directory $(CC_TEST_WORKSPACE_ROOT) \
+  --port `cat "$(CC_TEST_WORKSPACE_ROOT)/serverport"` --stop; \
+  rm -f "$(CC_TEST_WORKSPACE_ROOT)/serverport"; \
+  HOME="$(CC_TEST_WORKSPACE_ROOT)" ${CODECHECKER_CMD} server -l
 
 # Preserve the error status of the previous command but always be able to
 # shut down servers.
 EXIT_ERROR = { ${SHUTDOWN_SERVER_CMD}; exit 1; }
 
 FUNCTIONAL_TEST_CMD = $(REPO_ROOT) BUILD_DIR=$(BUILD_DIR) $(CLANG_VERSION) $(TEST_PROJECT) \
-		nosetests $(NOSECFG) tests/functional \
-		&& { ${SHUTDOWN_SERVER_CMD}; } || ${EXIT_ERROR}
+  nosetests $(NOSECFG) tests/functional \
+  && { ${SHUTDOWN_SERVER_CMD}; } || ${EXIT_ERROR}
 
 MAKE_DB_CMD = bash -c 'psql -h localhost \
-		-p $${TEST_DBPORT} -U $${TEST_DBUSERNAME} postgres \
-		-c "CREATE DATABASE codechecker_config_$${CODECHECKER_DB_DRIVER}"'
+  -p $${TEST_DBPORT} -U $${TEST_DBUSERNAME} postgres \
+  -c "CREATE DATABASE codechecker_config_$${CODECHECKER_DB_DRIVER}"'
 
 DROP_DB_CMD = bash -c 'psql -h localhost \
-		-p $${TEST_DBPORT} -U $${TEST_DBUSERNAME} postgres \
-		-c "DROP DATABASE IF EXISTS codechecker_config_$${CODECHECKER_DB_DRIVER}"'
+  -p $${TEST_DBPORT} -U $${TEST_DBUSERNAME} postgres \
+  -c "DROP DATABASE IF EXISTS codechecker_config_$${CODECHECKER_DB_DRIVER}"'
+
+run_test:
+	$(ACTIVATE_DEV_VENV) && \
+	$(REPO_ROOT) BUILD_DIR=$(BUILD_DIR) $(CLANG_VERSION) $(TEST_PROJECT) \
```

Comment:
Both `run_test` and `run_test_in_env` targets are using this part of code. This should be also refactored.